float64[3] posError
float64[3] velError
float64[3] Error2
float64[3] Error2_0
float64[3] intergral
float64[3] miu_t
float64[3] templesgn
float64[3] templemiu
float64    data1
float64    data2
float64    data3
float64    data4
